testsuite/features/step_definitions/command_steps.rb

```diff
@@ -603,11 +603,69 @@
   raise "Not found:\n#{output}" unless output.include?(text)
 end
 
+Then(/^tftp should be running on the proxy$/) do
+  node = get_target('proxy')
+  case $product
+  # TODO: Should we handle this in Sumaform?
+  when 'Uyuni'
+    step %(I enable repositories before installing branch server)
+    cmd = 'zypper --non-interactive --ignore-unknown remove atftp && ' \
+          'zypper --non-interactive install tftp && ' \
+          'systemctl enable tftp.socket && ' \
+          'systemctl start tftp.socket'
+    node.run(cmd)
+    step %(I disable repositories after installing branch server)
+  else
+    cmd = "systemctl enable tftp.socket && systemctl start tftp.socket"
```

Comment:
As you run this in all cases, maybe you could move it outside the `when`

(yeah, it would make two calls to `run` instead of one, but that's okay)